### PR TITLE
CM 367 return objects for errors (and small readme update)

### PR DESCRIPTION
**Pull request by seanmajorpayne**
Updating the errors so that they return objects as Nick requested

Added PDB information to readme

### CODE REVIEW

**Comment by y-himanen** [comment]
this won't work with what I've written because the store_subscription_data function needs to be edited too. let's chat and figure out the best way to do this.

File: knowledge_graph/routes.py [hunk 1]
```diff
@@ -314,64 +278,70 @@ def get_feed():
...
-    recommended_general_myths = MYTH_PROCESSOR.get_user_general_myth_nodes()
-    climate_general_myths = {"myths": recommended_general_myths}
-    return jsonify(climate_general_myths), 200
+    try:
+        recommended_general_myths = MYTH_PROCESSOR.get_user_general_myth_nodes()
+        climate_general_myths = {"myths": recommended_general_myths}
+        return jsonify(climate_general_myths), 200
+    except:
+        error = {"error": "Failed to Process Myths"}
+        return make_response(error), 400
```
Comment (NickCallaghan) [inline knowledge_graph/routes.py:333]: We should return a 500 here as its a server problem

**Review by NickCallaghan — APPROVED** [review]

**Comment by y-himanen** [comment]
do not merge this, please

**Comment by y-himanen** [comment]
changes will be incorporated elsewhere

**Pull request closed without merging**